schemas/schema-0.0.0.json

```diff
@@ -0,0 +1,249 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "CMS Code.json Metadata",
+    "description": "A metadata standard for software repositories of CMS",
+    "type": "object",
+    "properties": {
+      "items": {
+          "name": {
+              "type": "string",
+              "description": "Name of the project or software"
+          },
+          "softwareDescription": {
+              "type": "object",
+              "properties": {
+                  "en": {
+                      "type": "object",
+                      "description": "Language of description (English)",
+                      "properties": {
+                          "shortDescription": {
+                              "type": "string", 
+                              "description": "A short description of the project. It should be a single line containing a single sentence. Maximum 150 characters are allowed.",
+                              "maxLength": 150
+                          },
+                          "longDescription": {
+                              "type": "string", 
+                              "description": "Provide longer description of the software, between 150 and 10000 chars. It is meant to provide an overview of the capabilities of the software for a potential user.",
+                              "minLength": 100
+                          }
+                      }
+                  },
+                  "required": ["shortDescription", "longDescription"]
+              },
+              "required": ["en"]
+          },
+          "status": {
+              "type": "string",
+              "enum": ["Ideation", "Development", "Alpha", "Beta", "Release Candidate", "Production", "Archival"],
+              "description": "Development status of the project"
+          },
+          "permissions": {
+              "type": "object",
+              "description": "An object containing description of the usage/restrictions regarding the release",
+              "properties": {
+                  "license": {
+                      "type": "array",
+                      "items": {
+                          "type": "object",
+                          "properties": {
+                              "url": {
+                                  "type": "string",
+                                  "format": "uri",
+                                  "description": "The URL of the release license"
+                              },
+                              "name": {
+                                  "type": "string",
+                                  "enum": ["CC0-1.0", "Apache-2.0", "MIT", "MPL-2.0", "GPL-2.0-only", "GPL-3.0-only", "GPL-3.0-or-later", "LGPL-2.1-only", "LGPL-3.0-only", "BSD-2-Clause", "BSD-3-Clause", "EPL-2.0", "Other"],
```

Comment:
let's also add "none" here at the end. That will give us good license compliance fodder later.